src/app/containers/ProposePage/index.tsx

```diff
@@ -0,0 +1,214 @@
+import React, { useState } from 'react';
+import {
+  governance_proposalThreshold,
+  governance_propose,
+  governorType,
+} from '../BlockChainProvider/requests/governance';
+import { isAddress } from 'web3-utils';
+import { useHistory } from 'react-router-dom';
+import { Icon } from '@blueprintjs/core';
+import { useAccount } from 'app/hooks/useAccount';
+import { FormSelect } from 'app/components/FormSelect';
+import { useStaking_getCurrentVotes } from 'app/hooks/staking/useStaking_getCurrentVotes';
+import { bignumber } from 'mathjs';
+import { fromWei } from '../../../utils/helpers';
+import { toastError } from 'utils/toaster';
+
+const initRow = {
+  target: '',
+  value: '',
+  signature: '',
+  calldata: '',
+};
+export function ProposePage() {
+  const history = useHistory();
+  const [rows, setRows] = useState([initRow]);
+  const [description, setDescription] = useState('');
+  const [governor, setGovernor] = useState<governorType>('governorAdmin');
+  const account = useAccount();
+  const votes = useStaking_getCurrentVotes(account);
+
+  const updateRow = (value, field, rowIndex) => {
+    const newRows = [...rows.map(row => ({ ...row }))];
+    newRows[rowIndex][field] = value;
+    setRows(newRows);
+  };
+  const invalidRows = () =>
+    !!rows.find(
+      row =>
+        !isAddress(row.target) || !row.value || !row.signature || !row.calldata,
```

Comment:
Always transform wallet addresses to lowercase because isAddress and checksum checks fails when address is copied from wallet of wrong chain id.
```suggestion
        !isAddress(String(row.target).toLowerCase()) || !row.value || !row.signature || !row.calldata,
```

Reply:
Fixed.